### PR TITLE
Add Harness source and Faros destination's converter

### LINKED ISSUE #152: Add Harness source and Faros converter
Add a Harness source and converter based on [Harness](https://github.com/faros-ai/feeds/tree/master/feeds/cicd/harness-feed) feed

### PR DESCRIPTION
**Pull request by Roma-Kyrnis**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Related issues

> Fix #152 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: sources/harness-source/resources/spec.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "properties": {
+      "api_key": {
+        "type": "string",
+        "title": "Harness API key",
+        "airbyte_secret": true
+      },
+      "account_id": {
+        "type": "string",
+        "title": "Harness account ID",
+        "airbyte_secret": true
```
Comment (cjwooo) [inline sources/harness-source/resources/spec.json:21]: Does the account id need to be secret?

**Comment by cjwooo** [comment]
Is there a test Harness server we can spin up to test, similar to how we do Jenkins, or do we need to test against a live Harness account?

File: destinations/faros-destination/src/converters/harness/executions.ts [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+    } else {
+      return undefined;
+    }
+  }
+
+  private parseWorkflow(
+    execution: HarnessExecutionNode,
+    source: string
+  ):
+    | {
```
Comment (cjwooo) [inline destinations/faros-destination/src/converters/harness/executions.ts:146]: This seems pretty clunky read. Can you make the non-undefined return type a local interface?

File: sources/harness-source/resources/spec.json (hunk 1 above)
Comment (Roma-Kyrnis) [reply sources/harness-source/resources/spec.json:21]: Account id can be public. I'll change it to not be a secret

File: destinations/faros-destination/src/converters/harness/executions.ts (hunk 2 above)
Comment (Roma-Kyrnis) [reply destinations/faros-destination/src/converters/harness/executions.ts:146]: Ok. I've created interface

**Comment by Roma-Kyrnis** [comment]
Harness has [delegate](The Harness Delegate is a service). It's service but not Harness's local instance

**Review by cjwooo — APPROVED** [review]

**Comment by tovbinm** [comment]
Yay!

Please update README

**Pull request merged**